helper/helics-helper.cc

```diff
@@ -30,19 +30,17 @@ HelicsHelper::HelicsHelper()
 }
 
 void
-HelicsHelper::SetupFederate(void)
-{
-  helics::FederateInfo fi (name);
-  fi.coreType = helics::coreTypeFromString (core);
-  //fi.timeDelta = timedelta;
-  fi.timeDelta = helics::loadTimeFromString ("1ns");
-  if (!coreinit.empty()) {
-    fi.coreInitString = coreinit;
-  }
-  helics_federate = std::make_shared<helics::MessageFederate> (fi);
+HelicsHelper::SetupFederate(void) {
+    helics::FederateInfo fi{};
+    fi.coreType = helics::coreTypeFromString(core);
+    fi.setProperty(helics_property_time_delta, helics::loadTimeFromString("1ns"));
+    if (!coreinit.empty()) {
+        fi.coreInitString = coreinit;
+    }
+    helics_federate = std::make_shared<helics::MessageFederate>(name, fi);
 }
 
-// Recognized args:
+// Recognized args: FIXME: some of these no longer seem to work.
```

Comment:
We may want to make sure we have correctly identified the handled argument types for this or remove this section of comments.